src/app/components/nav/header-static/header-static.component.scss

```diff
@@ -194,7 +194,7 @@ header {
     margin-top: 15px;
     margin-left: 0;
     &.at-home {
-      margin-left:0px;
+      margin-left:0;
```

Comment:
What was the problem with `px`?

Reply:
It is not a error but a CSS linting problem.

Reply:
There is no difference bwtween `0` and `0px`, both work the same. It is only meant for better readability.